src/state/slices/opportunitiesSlice/resolvers/rFOX/index.ts

```diff
@@ -59,6 +59,8 @@ export const rFOXStakingMetadataResolver = async ({
     .times(stakingAssetMarketData.price)
     .toFixed(2)
 
+  const underlyingAssetIds = [foxOnArbitrumOneAssetId]
```

Comment:
revert?

Reply:
If I revert this, TS is not happy because it says `[foxOnArbitrumOneAssetId]` is `readonly AssetId[]` 

Happy with ever this or casting, as you want

Reply:
Ohh, let's leave it as-is then!

Reply:
Happy with both casting or letting this one as natural type

I can't revert without casting because its considered as `readonly AssetId[]` without this, as you prefer